br/pkg/restore/import_mode_switcher.go

```diff
@@ -47,8 +48,8 @@ func NewImportModeSwitcher(
 }
 
 // switchToNormalMode switch tikv cluster to normal mode.
-func (switcher *ImportModeSwitcher) switchToNormalMode(ctx context.Context) error {
-	close(switcher.switchCh)
+func (switcher *ImportModeSwitcher) SwitchToNormalMode(ctx context.Context) error {
+	sync.OnceFunc(func() { close(switcher.switchCh) })
```

Comment:
This is a strange behavior exists a long time. in stream restore `RestorePreWork` will skip the switch tikv mode, but in `RestorePostWork` it will switch back to normal.  so this sync.OnceFunc make us don't close channel twice. 
I can change this behavior in next PR.  I think that will be more clear.

Reply:
Well, but for now, this line of code is actually a nop...